pkg/client/clientset_generated/clientset/scheme/register.go

```diff
@@ -17,14 +17,15 @@ limitations under the License.
 package scheme
 
 import (
+	os "os"
```

Comment:
This is gofmt only.  Do you want this in this commit?

Reply:
I'll do a PR that updates apimachinery

Reply:
https://github.com/kubernetes/kops/pull/4473